libraries/stdlib/src/kotlin/collections/Maps.kt

```diff
@@ -21,15 +21,18 @@ private object EmptyMap : Map<Any, Any> {
 /** Returns an empty read-only map of specified type */
 public fun emptyMap<K, V>(): Map<K, V> = EmptyMap as Map<K, V>
 
-/** Returns a new read-only map of given pairs, where the first value is the key, and the second is value */
+/**
+ * Returns a new read-only map with the specified contents, given as a list of pairs
+ * where the first value is the key and the second is the value.
+ */
```

Comment:
specify, that it behaves as sequential `put` calls, so that if conflicting keys exist in source sequence, last one wins.